src/components/Notifications/NotificationsFeedTabs.tsx

```diff
@@ -0,0 +1,167 @@
+import React, { useCallback } from "react"
+import { Tab, Tabs } from "@mui/material"
+import { i18n } from "./NotificationsFeedTabs.i18n"
+import {
+  DCLNotificationProps,
+  NotificationActiveTab,
+  NotificationLocale,
+} from "./types"
+import { NotificationComponentByType } from "./utils"
+import { EmptyInboxIcon, HistoryIcon } from "../Icon"
+import { NotificationsFeedTabsProps } from "./NotificationsFeedTabs.types"
+import {
+  EmptyViewContainer,
+  EmptyViewDescription,
+  EmptyViewTitle,
+} from "./NotificationsFeed.styled"
+import {
+  NotificationFeedPrevious,
+  NotificationFeedTabsContent,
+  NotificationFeedTabsHeader,
+  NotificationFeedTabsListContainer,
+  NotificationFeedTabsListWrapper,
+  NotificationFeedTabsTitle,
+} from "./NotificationsFeedTabs.styled"
+
+const NotificationHandler = ({
+  locale,
+  notification,
+  renderProfile,
+}: {
+  notification: DCLNotificationProps
+  locale: NotificationLocale
+  renderProfile?: (address: string) => JSX.Element | string | null
+}) => {
+  const NotificationComponent = NotificationComponentByType[notification.type]
+  return (
+    <NotificationComponent
+      notification={notification}
+      locale={locale}
+      renderProfile={renderProfile}
+    />
+  )
+}
+
+const NoReadNotifications = ({ locale }: { locale: NotificationLocale }) => (
+  <EmptyViewContainer>
+    <HistoryIcon sx={{ fontSize: "115px" }} />
+    <EmptyViewTitle>{i18n[locale].feed.history.title}</EmptyViewTitle>
+    <EmptyViewDescription>
+      {i18n[locale].feed.history.description}
+    </EmptyViewDescription>
+  </EmptyViewContainer>
+)
+
+const NoNotifications = ({ locale }: { locale: NotificationLocale }) => (
+  <EmptyViewContainer>
+    <EmptyInboxIcon sx={{ fontSize: "115px" }} />
+    <EmptyViewTitle>{i18n[locale].feed.empty.title}</EmptyViewTitle>
+    <EmptyViewDescription>
+      {i18n[locale].feed.empty.description}
+    </EmptyViewDescription>
+  </EmptyViewContainer>
+)
+
+const NotificationFeedTabs = React.memo((props: NotificationsFeedTabsProps) => {
+  const {
+    unreadNotifications,
+    locale,
+    previousNotifications,
+    readNotifications,
+    activeTab,
+    isModal,
+    renderProfile,
+    onChangeTab,
+  } = props
+
+  const onChangeHandler = useCallback(
+    (e, tab: NotificationActiveTab) => {
+      onChangeTab(e, tab)
+    },
+    [onChangeTab]
+  )
+  return (
+    <>
+      <NotificationFeedTabsHeader>
+        <NotificationFeedTabsTitle>
+          {i18n[locale].feed.title}
+        </NotificationFeedTabsTitle>
+      </NotificationFeedTabsHeader>
+      <NotificationFeedTabsContent>
+        <Tabs
+          value={activeTab}
+          onChange={onChangeHandler}
+          indicatorColor="primary"
+        >
+          <Tab
+            label={i18n[locale].feed.tabs.newest}
+            value={NotificationActiveTab.NEWEST}
+            onClick={(e) => onChangeTab(e, NotificationActiveTab.NEWEST)}
```

Comment:
There's an `onChangeHandle` method for the `Tabs` component and a `onClick` method for the `Tab`, is it using both?

Reply:
good catch